pymc3/distributions/multivariate.py

```diff
@@ -524,6 +518,38 @@ def logp(value, n, p):
         )
 
 
+class DirichletMultinomialRV(RandomVariable):
+    name = "dirichlet_multinomial"
+    ndim_supp = 1
+    ndims_params = [0, 1]
```

Comment:
Not sure about this

Reply:
Seems fine, but you might need to copy these lines to have the op even initialize: https://github.com/aesara-devs/aesara/blob/3a3adaee3b8ceccff6d0208996f117fcd5e5eb6f/aesara/tensor/random/basic.py#L518-L521

After copying that I can run the following snipped successfully:

```python
from pymc3.distributions.multivariate import dirichlet_multinomial
dirichlet_multinomial(5, [1, 1, 1, 1])
dirichlet_multinomial(5, [1, 1, 1, 1]).eval()
# array([2, 2, 1, 0])
```

Reply:
Oooh thanks, that's a sneaky one! That was driving me crazy. It works for me too now 🎉